packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -206,139 +206,166 @@ export interface GridOptions {
   /**
    * Callback fired when the edit cell value changes.
    * @param {GridEditCellPropsParams} params With all properties from [[GridEditCellPropsParams]].
-   * @param {React.SyntheticEvent} event The event that caused this prop to be called.
+   * @param {MuiEvent} event The event that caused this prop to be called.
```

Comment:
```suggestion
   * @param {MuiEvent<React.SyntheticEvent>} event The event that caused this prop to be called.
```

? x the other

---

I would personally be fine if I had to do this in my codebase.:

```jsx
(event as any).defaultMuiPrevented = true;
```

```suggestion
   * @param {React.SyntheticEvent} event The event that caused this prop to be called.
```

but I can understand if we don't like this idea 😆

Reply:
We started to use this `MuiEvent`. Would be nice to keep it, because of the special property. Sometimes using it can lead to terrible typings when the event can be fired both by a synthetic event as well as by a native event, e.g. `MuiEvent<React.MouseEvent | DocumentEventMap['click']>`.